### ISSUE TITLE
ERR_INVALID_ARG_TYPE when starting as daemon (systemd)

### ISSUE THREAD

**Issue by Qomaa** [xemle/home-gallery#80]
Hi,

I start the home gallery server as a daemon (service) with systemd.
Since 1.9.0 I get an error at the start. [Looking at the home gallery code](https://github.com/xemle/home-gallery/blob/54f74940fc1492e54db847f1d242419d646d2877/packages/cli/src/config/find-config.js#L14), it turns out that the HOME environment variable is not set.

That is because some environment variables are not set when a systemd service is running: [https://serverfault.com/a/910166](https://serverfault.com/a/910166)

**I solved my problem by adding User=root in Service config.**

`sudo systemctl edit homegallery.service`

```
[Service]
User=root
```

Maybe someone can make the code a more robust at that point (if it's possible): Check if a HOME environment variable is set before join the path.

The error I receive:

```
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]: node:internal/validators:163
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     ^
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at new NodeError (node:internal/errors:399:5)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at validateString (node:internal/validators:163:11)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at Object.join (node:path:1172:7)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at Object.<anonymous> (/tmp/caxa/home-gallery/master/home-gallery/node_modules/@home-gallery/cli/dist/config/find-config.js:14:99)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at Module._compile (node:internal/modules/cjs/loader:1254:14)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at Module.load (node:internal/modules/cjs/loader:1117:32)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at Module._load (node:internal/modules/cjs/loader:958:12)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at Module.require (node:internal/modules/cjs/loader:1141:19)
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:     at require (node:internal/modules/cjs/helpers:110:18) {
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]:   code: 'ERR_INVALID_ARG_TYPE'
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]: }
May 28 09:43:16 m home-gallery-latest-linux-x64[4040458]: Node.js v18.15.0

```

**Comment by xemle** [comment]
Hi @Qomaa 

thank you for finding and reporting this issue. Would you mind to provide an PR for it, that would be great.

Otherwise provide an HOME env in your setting and subsribe this repo to get notified, when this issue will be fixed.

**Comment by xemle** [comment]
Fixed in v1.11.0, precisely in 5a3fca4. HOME env is not required anymore. 

See further changes on v1.11.0 in the [Changelog](https://docs.home-gallery.org/changelog.html)
